acceptance/auth/credentials/basic/script

```diff
@@ -0,0 +1,9 @@
+# Unset the token which is configured by default
+# in acceptance tests
+export DATABRICKS_TOKEN=""
+
+# "username:password" in base64 is dXNlcm5hbWU6cGFzc3dvcmQ=, expect to see this in Authorization header
```

Comment:
The comment helps, but the trace can still inadvertently leak something and go undetected.

You could include the base64 encoded version of the string as a replacement in the configuration, then it will show up as `[ENCODED_AUTH]` or something, and fail if it is anything else.